### ISSUE TITLE
[Infrastructure]: <title>Is the Enarx project still being actively maintained and updated?

### ISSUE THREAD

**Issue by digitalinformation** [enarx/enarx#2543]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Description

Is the Enarx project still being actively maintained and updated?

**Comment by rjzak** [comment]
It's in a bit of a hiatus at the moment, hopefully not for too long. PRs welcome! :)

**Comment by rjzak** [comment]
Yes! I continue to work on the project. I've been working on the `flagset`, `ciborium`, and `sgx` crates, mostly in terms of reviewing PRs from the community, updating dependencies, and doing the occasional release.

Currently I'm working with @Freax13 to get SNP V10 patches https://github.com/enarx/enarx/pull/2552 working, with the blocker being getting the AMD SEV-SNP CI runner working again. The SGX runner has a permissions issue when running one of the unit tests, which I'm also investigating.

Once the CI runners are functional again, I plan to go through and update all the dependencies and try to clean up some of the PR backlog.

I've also been getting some emails and been in some meetings around attestation, and has resulted in me getting some things out of my head into into emails, which I need to make into documentation :)

Since the closure of Profian (in no particular order):
* Some instructional videos on YouTube were inadvertently deleted, I think @nickvidal was seeing about getting some of them back. I think he has the transcripts at least.
* Some of the supporting crates are accepting community PRs, so we want to be useful to the wider community even if the development of those crates aren't supporting Enarx right now.
* We don't have a long term technical direction at the moment, we're working on that.
* We want to get a public Steward set up so people can experiment with attestation with their own hardware.
* We have PRs to have Steward https://github.com/profianinc/steward/pull/315 and Drawbridge https://github.com/profianinc/drawbridge/pull/519 relicensed, and we'll move them to the Enarx organisation after.

In summary, we're here, we're interested, Enarx isn't going anywhere; but development isn't going to be as fast as before (for now at least). I'm happy to get on a call to discuss further if that's of interest.

**Comment by mh4ck-Thales** [comment]
That's very good news ! I'm interested to help, I've been studying the docs and the project seems overall very interesting. If https://chat.enarx.dev is still the way to go to discuss about Enarx, I'll head here and explain further my interests and why I'm interested about Enarx. A call could also be useful.

**Comment by rjzak** [comment]
That's still the chat location, and I think some folks to hang out there, or at least respond when tagged.
